ArduCopter/heli.cpp

```diff
@@ -183,6 +182,20 @@ void Copter::heli_update_rotor_speed_targets()
                 motors.set_desired_rotor_speed(0);
             }
             break;
+        case ROTOR_CONTROL_MODE_CLOSED_LOOP_POWER_OUTPUT:
+            // pass setpoint through as desired rotor speed
+            if (rsc_control_deglitched > 0) {
+                motors.set_interlock(true);
+                if ((rpm_sensor.get_rpm(0) > 0) && ((heli_flags.using_governor_switch && heli_flags.governor_enable) || !heli_flags.using_governor_switch)){
+                    motors.set_rsc_governor_enabled(true, motors.get_gov_rpm_setpoint(), rpm_sensor.get_rpm(0));
```

Comment:
Possibly...

So one thing about this, when I set out to do this, I had asked Randy and Tridge if we should re-use the RSC_Setpoint param to mean RPM in this instance, while it is used to mean PWM in the other RSC Modes. However, they felt this was too confusing.  So I added the new Gov RPM param.

And so, it seems to fall out from that, that we would use a new function to set it?  It does seem to be cleaner and easier to understand.  It's just a bit more code and Flash memory as the disadvantage?